src/Storage/Service.ts

```diff
@@ -7,11 +7,14 @@ import { childWithFileName } from 'Helpers/Logging'
 import { ClaimController } from './ClaimController'
 import { ServiceConfiguration } from './ServiceConfiguration'
 
+const toMinutes = (milliseconds: number) => 1000 * 60 * milliseconds
```

Comment:
Avoid adding such general purpose functions in files dedicated to domain specific things. 

Add them to /Helpers if you think they could be reused, or declare them inside the function that's using them if it's only for readability.